contributors/devel/release/patch_release.md

```diff
@@ -0,0 +1,87 @@
+# Building Kubernetes patch releases
+
+This document describes the process for creating Kubernetes patch releases and communicating publishing
+status regarding known issues impacting the current release.
+
+## Process
+
+After a Kubernetes minor release, patch releases are managed by the Kubenretes patch release czar.
```

Comment:
nit: Do we still use the `czar` term? I thought we had switched to `manager`...?
Not very important though